### PR TITLE
Add husky & Pretty Quick

### PR DESCRIPTION
**Pull request by lawweiliang**
@elie222 ,

As disccussed earlier, husky and pretty-quick was added.

There are some changes over husky side.
Previously, it will automatically install the git hook after finished `npm install`. Now, if it is a fresh clone project, we need to run `npm run prepare` manually to install the git hook. (Need to include this over readme file)

Here is the articles that explain why husky doesn't autoinstall anymore?
https://blog.typicode.com/husky-git-hooks-autoinstall/

---

Update Readme file on 'How to Install ERC721R'

😄 😄 😄

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,15 @@
 # ERC721R
 
+## How to Install
+
+```
+git clone https://github.com/exo-digital-labs/ERC721R.git
```
Comment (elie222) [inline README.md:6]: i think we should remove this line. if you have this on your computer the steps are just `npm install`

File: README.md [hunk 2]
```diff
@@ -1,5 +1,15 @@
...
+
+cd ERC721R
```
Comment (elie222) [inline README.md:8]: and this line too

Comment (lawweiliang) [reply README.md]: Noted

**Comment by elie222** [comment]
@lawweiliang https://github.com/exo-digital-labs/ERC721R/issues/36

**Pull request merged**